materialized_sql_view/views/materialized_sql_view.xml

```diff
@@ -25,6 +25,10 @@
                                 string="Refresh now"
                                 type="object" states="nonexistent,refreshed,aborted"
                                 groups="materialized_sql_view.manager_group"/>
+	                    <button name="refresh_materialized_view"
+	                            string="Force Refresh"
+	                            type="object" states="refreshing"
+	                            groups="materialized_sql_view.manager_group"/>
```

Comment:
Hello, this button will run the refresh process immediately,  "Refresh now" button will only ask the job to run it.